incubation.md

```diff
@@ -0,0 +1,79 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which new Jupyter Subprojects are created and evaluated for inclusion in the main Jupyter organization. In this context a Subproject is a GitHub repository with a development team that follows the Jupyter governance and contribution model. The goals of the incubation process are:
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while complying with corporate contribution restrictions.
+* Contributors can work with the community and gather feedback that will help them develop and refine a clear and concise integration proposal.
+* The broader user community can easily distinguish between stable core project feature functions and what is on the docket for potential future integration.
+* Enables the core development team to assess the readiness an applicability of proposed technologies against a backdrop of the broader project design efforts. 
+* Allows the core team to re-evaluate existing jupyter org repos to determine if any need to be moved to jupyter-incubator.
```

Comment:
Assuming the jupyter-incubator org was active and a process approved. At that point we could take a hard look at the repos in the main jupiter org and ask "Are these repos technologies under incubation or are they well integrated and complimentary repos for the stable mainline project?" I feel the jupyter-incubator org gives us a mechanism to help organize the project. Your question really pertains to what is the criteria for approving any idea as a new mainline jupyter org repo?